packages/cli/src/commands/init/init.js

```diff
@@ -90,6 +93,87 @@ async function createFromExternalTemplate(
   }
 }
 
+async function installPods(loader: typeof Ora) {
+  process.chdir('ios');
+
+  const hasPods = await fs.pathExists('Podfile');
+
+  if (!hasPods) {
+    return;
+  }
+
+  try {
+    await commandExists('pod');
+  } catch (err) {
+    loader.succeed();
+
+    const {shouldInstallCocoaPods} = await inquirer.prompt([
+      {
+        type: 'confirm',
+        name: 'shouldInstallCocoaPods',
+        message: 'CocoaPods is not installed, do you want to install it?',
+      },
+    ]);
+
+    if (shouldInstallCocoaPods) {
+      try {
+        // First attempt to install `cocoapods`
+        await execa('gem', ['install', 'cocoapods'], {
+          stdio: 'pipe',
+        });
```

Comment:
@thymikee, I know you suggested installing WITHOUT `sudo`, but according to CocoaPods installation guide, users will need to modify the $PATH themselves in order to make this one work. 

Otherwise `pod` command will not be available.

I'd just go with `sudo` for those who don't have it to make this easier. It's still recommended way.

Reply:
I'm pretty sure it's automatically handled when you install ruby with brew

Reply:
I think the default `ruby` that most people have is the one that comes with a Mac. For example, that's the one I am using myself.

Anyway, let's check this. We need to be 100% sure that we are shipping the best experience possible here.

CC: @orta, what do you think?

Reply:
I think this is reasonable, editing the `$PATH` overall is definitely preferable in the long run - but we do recommend using `sudo` in a few places because it works and doesn't affect the end results of a pod installation (in terms of permissions etc). I'm actually on a fresh computer so this is what it'd look like under the hood:

```
Last login: Tue May  7 09:39:43 on ttys002
Ortas-MacBook-Pro:~ orta$ ruby -v
ruby 2.3.7p456 (2018-03-28 revision 63024) [universal.x86_64-darwin18]

Ortas-MacBook-Pro:~ orta$ pod install
-bash: pod: command not found

Ortas-MacBook-Pro:~ orta$ gem install cocoapods
Fetching: ruby-macho-1.4.0.gem (100%)
ERROR:  While executing gem ... (Gem::FilePermissionError)
    You don't have write permissions for the /Library/Ruby/Gems/2.3.0 directory.

Ortas-MacBook-Pro:~ orta$ sudo gem install cocoapods
Password:
Fetching: ruby-macho-1.4.0.gem (100%)
Successfully installed ruby-macho-1.4.0
Fetching: nap-1.1.0.gem (100%)
Successfully installed nap-1.1.0
Fetching: gh_inspector-1.1.3.gem (100%)
Successfully installed gh_inspector-1.1.3
Fetching: fourflusher-2.2.0.gem (100%)
...
Parsing documentation for cocoapods-1.6.1
Installing ri documentation for cocoapods-1.6.1
Done installing documentation for ruby-macho, nap, gh_inspector, fourflusher, escape, colored2, concurrent-ruby, i18n, thread_safe, tzinfo, activesupport, atomos, CFPropertyList, claide, nanaimo, xcodeproj, molinillo, cocoapods-try, netrc, cocoapods-trunk, cocoapods-stats, cocoapods-search, cocoapods-plugins, cocoapods-downloader, cocoapods-deintegrate, fuzzy_match, cocoapods-core, cocoapods after 17 seconds
28 gems installed

Ortas-MacBook-Pro:~ orta$ pod -v
[!] Unknown command: `-v`
Did you mean: env?

Usage:

    $ pod COMMAND

      CocoaPods, the Cocoa library package manager.

Commands:

    + cache         Manipulate the CocoaPods cache
    + deintegrate   Deintegrate CocoaPods from your project
    + env           Display pod environment
    + init          Generate a Podfile for the current directory
    + install       Install project dependencies according to versions from a
                    Podfile.lock
    + ipc           Inter-process communication
    + lib           Develop pods
    + list          List pods
    + outdated      Show outdated project dependencies
    + plugins       Show available CocoaPods plugins
    + repo          Manage spec-repositories
    + search        Search for pods
    + setup         Setup the CocoaPods environment
    + spec          Manage pod specs
    + trunk         Interact with the CocoaPods API (e.g. publishing new specs)
    + try           Try a Pod!
    + update        Update outdated project dependencies and create new Podfile.lock

Options:

    --silent        Show nothing
    --version       Show the version of the tool
    --verbose       Show more debugging information
    --no-ansi       Show output without ANSI codes
    --help          Show help banner of specified command
```

Which is fine.

Reply:
Shall I keep only with `sudo` then?

Reply:
I think it's fine like this. @orta just demonstrated an example where he did run sudo-less first and with sudo second time, so let's leave it.

Reply:
Yep, prefer the sudo-less one but expect it to fail